packages/webpack/src/config/base.js

```diff
@@ -19,6 +20,14 @@ import WarningIgnorePlugin from '../plugins/warning-ignore'
 
 import { reservedVueTags } from '../utils/reserved-tags'
 
+function maybeResolve (name) {
```

Comment:
Nice util too! maybeMoveToUtils? :D

Reply:
Done. I've also renamed it to `tryResolve` for more consistency. 👍